src/source/Crypto/IOBuffer.c

```diff
@@ -0,0 +1,101 @@
+#define LOG_CLASS "IOBuffer"
```

Comment:
CamelCase the name of the file and the names. Something like IoBuffer.c

The APIs will then be createIoBuffer

Reply:
I think it's better to be "IO", input output, since they're not full words. If "Io", it sound like as if there's a word "io".

Reply:
OK